beaver/transports/sqs_transport.py

```diff
@@ -81,15 +86,18 @@ def callback(self, filename, lines, **kwargs):
         return True
 
     def _send_message_batch(self, message_batch):
-        try:
-            result = self._queue.write_batch(message_batch)
-            if not result:
-                self._logger.error('Error occurred sending messages to SQS queue {0}. result: {1}'.format(
-                    self._queue_name, result))
-                raise TransportException('Error occurred sending message to queue {0}'.format(self._queue_name))
-        except Exception, e:
-            self._logger.exception('Exception occurred sending batch to SQS queue')
-            raise TransportException(e.message)
+        for queue in self._queue:
```

Comment:
Won't this queue the `message_batch` to every queue? Don't we only want to post to a new queue if the previous one is full?